integration/docker/Dockerfile

```diff
@@ -9,18 +9,6 @@
 # See the NOTICE file distributed with this work for information regarding copyright ownership.
 #
 
-# ARG defined before the first FROM can be used in FROM lines
-# Only 8 and 11 are supported.
-ARG JAVA_VERSION=8
-
-# Setup CSI
-FROM golang:1.15.13-alpine AS csi-dev
-ENV GO111MODULE=on
-RUN mkdir -p /alluxio-csi
-COPY ./csi /alluxio-csi
-RUN cd /alluxio-csi && \
-    CGO_ENABLED=0 go build -o /usr/local/bin/alluxio-csi
```

Comment:
Would you mind explaining why this part is not necessary anymore? Is this because we're migrating it to Dora?

Reply:
The CSI code will be moved to Alluxio/k8s-operator repo, and it is very likely to have its own image onwards